tcms/static/js/bugs.js

```diff
@@ -55,6 +55,9 @@ function fetchBugDetails(source, popover, cache = bugDetailsCache) {
     }
 
     jsonRPC('Bug.details', [source.href], data => {
+        if (!data) {
+            return
+        }
```

Comment:
Does anything break without this if statement when data is None (what is the equivalent of None in JavaScript) ? I think we're fine without this check.

Reply:
the equivalent of `None` in JS is `null`, `undefined`, `0`, `""`, etc. Every one of these values will return false and trigger the early exit in this case. Here, if the API returns `None` the result will be parsed as `undefined`. To answer your question, I am not sure if anything will break if we omit this check, will try it out